<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpWidgetClass</class>
 <widget class="QWidget" name="TcpWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Timeout:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ipEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="portEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="timeoutEdit"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTextEdit" name="dataEdit"/>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QHBoxLayout" name="sendLayout"/>
   </item>
   <item row="1" column="2" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
